core/serialisation/data_serialiser.py

```diff
@@ -306,6 +309,25 @@ class Meta:
     organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
 
 
+class ProgrammeManagementSchema(SQLAlchemySchema):
+    """Serialise ProgrammeManagement data"""
+
+    class Meta:
+        model = ProgrammeManagement
+
+    submission_id = auto_field(model=Submission, data_key="SubmissionID")
+    programme_id = auto_field(model=Programme, data_key="ProgrammeID")
+    payment_type = fields.String(attribute="data_blob.payment_type", data_key="PaymentType")
+    spend_for_reporting_period = fields.Float(
```

Comment:
Can we change all of the numbers to ```fields.Float``` when serialised out of the blob now? Or else we have some as ```Number``` and it's not clear why

Reply:
yeah, as above comment I'm fixing in a separate PR